sdk/tests/conformance2/context/methods-2.html

```diff
@@ -247,11 +249,7 @@
 "isSampler",
 "bindSampler",
 "samplerParameteri",
-"samplerParameteriv",
-"samplerParameteriv",
 "samplerParameterf",
-"samplerParameterfv",
-"samplerParameterfv",
```

Comment:
texParameter[if]v were not exposed in WebGL 1, and I can't recall any calls for adding them back. I think it makes sense to stay consistent with that. I can't imagine that texture border color extension is on any developer's wishlist. A convenience feature like that is worthless if it only works on a subset of platforms.